packages/adblocker/src/resources.ts

```diff
@@ -297,8 +297,18 @@ export default class Resources {
     }
   }
 
-  public getResource(name: string): { body: string; contentType: string; dataUrl: string } {
-    const { body, contentType } = this.resourcesByName.get(name) || getResourceForMime(name);
+  public getResource(name: string): {
+    name: string;
+    body: string;
+    contentType: string;
+    dataUrl: string;
+  } {
+    const resource = this.resourcesByName.get(name) || getResourceForMime(name);
```

Comment:
we have to add the `name` to `getResourceForMime` and the extension will have to generate WARs for those on the build time

Reply:
please add tests for that

Reply:
`getResourceForMime` is an import from `@remusao/small`. Enforcement should be there and a version bump from `small` will resolve this concern.